users/foldertypes.rst

```diff
@@ -35,3 +35,22 @@ Clicking this button will enforce this host's current state on the
 rest of the cluster. Any changes made to files will be overwritten by
 the version on this host, any files that don't exist on this host will
 be deleted, and so on.
+
+Receive Only Folder
+-------------------
+
+A folder can be set in "receive-only mode" among the folder settings.
+
+.. image:: folderreceiveonly.png
+
+In this mode, a device will never send any local updates to the cluster.
+It will only "mirror" the folder from the cluster.
```

Comment:
The use of the word "mirror" may confuse matters here. It may be better to say "It will only ever receive changes", if anything more than the first sentence.